lib/health_checks/rails.rb

```diff
@@ -16,17 +19,39 @@ def rails
 
   private
 
-  class RedisCheck < OkComputer::Check
-    def check
-      sidekiq_response = ::Sidekiq.redis(&:ping)
+  module Logger
```

Comment:
What's the use of this logger? Just to prepend `Error: ` at the beginning? Doesn't make much sense, that's what you have log levels like warn and error for

Reply:
So writing `Rails.logger.error` instead of `Logger.error` is too much for you? And that deserves creating a separate module? Come on

Reply:
yeah, whatever. Can't really say what is better

Reply:
Less code and simpler code is better